go.mod

```diff
@@ -4,8 +4,8 @@ go 1.14
 
 require (
 	github.com/aos-dev/go-integration-test/v3 v3.0.0
-	github.com/aos-dev/go-storage/v3 v3.6.0
-	github.com/dropbox/dropbox-sdk-go-unofficial v5.6.0+incompatible
+	github.com/aos-dev/go-storage/v3 v3.6.1-0.20210427074731-1b3b4e86c950
+	github.com/dropbox/dropbox-sdk-go-unofficial v1.0.1-0.20210420014012-0798489e838f
```

Comment:
The git version supports specifying `UploadSessionType` in `create_append` consistent with the document description. For `appender`，the `UploadSessionType` should be set `UploadSessionTypeSequential`, which is the default value.
I‘m not sure if it is neccesary to modify the version.

Reply:
While, as we intend to be `production ready`, I prefer to use stable release instead of tracking on git version.

Let's wait for response on https://github.com/dropbox/dropbox-sdk-go-unofficial/issues/77